avatar.css

```diff
@@ -0,0 +1,147 @@
+body{
+    background-color: rgb(235, 227, 227);
```

Comment:
Try to put colors and font size in style css file .i.e. you would be able to use these colors and fonts again as your stylesheet would be linked with your all the components